### ISSUE TITLE
[BUG] readme中提到的ericmodel.onnx 现在不能优化了

### ISSUE THREAD

**Issue by gttiankai** [daquexian/onnx-simplifier#76]
**Describe the bug**
```text
Simplifying...
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.7/site-packages/onnxsim/__main__.py", line 48, in <module>
    main()
  File "/usr/local/lib/python3.7/site-packages/onnxsim/__main__.py", line 36, in main
    args.input_model, check_n=args.check_n, perform_optimization=not args.skip_optimization, skip_fuse_bn=not args.enable_fuse_bn, input_shapes=input_shapes)
  File "/usr/local/lib/python3.7/site-packages/onnxsim/onnx_simplifier.py", line 308, in simplify
    model = optimize(model, skip_fuse_bn)
  File "/usr/local/lib/python3.7/site-packages/onnxsim/onnx_simplifier.py", line 231, in optimize
    onnx.checker.check_model(model)
  File "/usr/local/lib/python3.7/site-packages/onnx/checker.py", line 91, in check_model
    C.check_model(model.SerializeToString())
onnx.onnx_cpp2py_export.checker.ValidationError: 334 in initializer but not in graph input
```
**Model**
To reproduce the problem, please post download link of your model here, or send your model to daquexian566@gmail.com

**Comment by nihui** [comment]
torch.onnx._export(net, batch, "ericmodel.onnx", export_params=True, keep_initializers_as_inputs=True, opset_version=11)

**Comment by daquexian** [comment]
Thanks for your report! It has been solved in v0.2.10
